.travis.yml

```diff
@@ -1,31 +1,27 @@
-# Travis-CI file for Esp-Link
-
 language: c
-
 before_install:
-  - curl -Ls http://s3.voneicken.com/xtensa-lx106-elf-20160330.tgx | tar Jxf -
-  - curl -Ls http://s3.voneicken.com/esp_iot_sdk_v2.0.0.p1.tgx | tar -C .. -Jxf -
-
+- curl -Ls http://s3.voneicken.com/xtensa-lx106-elf-20160330.tgx | tar Jxf -
+- curl -Ls http://s3.voneicken.com/esp_iot_sdk_v2.0.0.p1.tgx | tar -C .. -Jxf -
 after_script:
```

Comment:
why all these formatting changes here? please don't make random formatting changes, it makes reviewing difficult, plus, I don't see why your formatting is better than mine...